### ISSUE TITLE
Enhancement idea

### ISSUE THREAD

**Issue by skylight102** [bayaraa/d2gl#19]
It would be super cool if you could scale the mini map smaller/larger via the "Ctrl + O" menu. I find when you use higher resolutions, the map gets a bit tiny, it's really difficult to see in certain areas, such as plethegon in PD2.

Keep up the good work!

**Comment by nelsonmenon** [comment]
taking advantage of the topic, I would also suggest to be able to choose the side of the mini map

**Comment by bayaraa** [comment]
Yeah mini map needs some improvements like zoom is not good on certain resoltions etc.
